### PR TITLE
Adds player-tracking into console for clients

### PR DESCRIPTION
**Pull request by ruuppa**
Bodged version, not great.

### CODE REVIEW

File: ProgrammingTask/client.py [hunk 1]
```diff
@@ -124,6 +124,9 @@ def main():
 
     microseconds = 0
 
+    p2Flag = False
```
Comment (je-l) [inline ProgrammingTask/client.py]: Small but important note, `p2Flag` is quite ambiguous name for an attribute. Maybe `player2_connected` or something like that?

**Review by je-l — CHANGES_REQUESTED** [review]
Small note about naming

File: ProgrammingTask/client.py (hunk 1 above)
Comment (je-l) [reply ProgrammingTask/client.py]: Please merge this after resolving how you see best!

Comment (ruuppa) [reply ProgrammingTask/client.py]: Good call, changed to be more descriptive

**Pull request merged**